chapters/en/chapter1/10.mdx

```diff
@@ -155,7 +154,7 @@ result = classifier("This is a course about the Transformers library")
 	]}
 />
 
-### 7. Select the sentence that best describes the terms "model," "architecture," and "weights."
+### 7. Select the sentence that best describes the terms "model", "architecture", and "weights".
```

Comment:
Nice catch!

Reply:
Hehe, thank you sir! Looking forward to more :)